Buildings/Experimental/DHC/CentralPlants/Cooling/Subsystems/Examples/CoolingTowerParallel.mo

```diff
@@ -80,67 +29,35 @@ model CoolingTowerParallel
-  connect(onOffCon.u,TSwi.y)
-    annotation (Line(points={{-12,-86},{-39,-86}},color={0,0,127}));
-  connect(TVol.T,onOffCon.reference)
-    annotation (Line(points={{-40,-40},{-30,-40},{-30,-74},{-12,-74}},color={0,0,127}));
-  connect(exp.ports[1],vol.ports[2])
-    annotation (Line(points={{80,-20},{30,-20}},color={0,127,255}));
-  connect(TSetLea.y,conFan.u_s)
-    annotation (Line(points={{-39,90},{-22,90}},color={0,0,127}));
-  connect(senTCWLvg.T,conFan.u_m)
-    annotation (Line(points={{40,61},{40,74},{-10,74},{-10,78}},color={0,0,127}));
-  connect(weaDat.weaBus,weaBus)
-    annotation (Line(points={{-80,90},{-70,90}},color={255,204,51},thickness=0.5));
-  connect(senTCWLvg.port_b,vol.ports[1])
-    annotation (Line(points={{50,50},{60,50},{60,-20},{27.3333,-20}},color={0,127,255}));
-  connect(conFan.y,cooTowPar.uFanSpe)
-    annotation (Line(points={{1,90},{6,90},{6,70},{-10,70},{-10,52},{-2,52}},color={0,0,127}));
-  connect(cooTowPar.port_b,senTCWLvg.port_a)
-    annotation (Line(points={{20,50},{30,50}},color={0,127,255}));
-  connect(vol.ports[3],pum.port_a)
-    annotation (Line(points={{32.6667,-20},{-60,-20},{-60,50},{-40,50}},color={0,127,255}));
-  connect(pum.port_b,cooTowPar.port_a)
-    annotation (Line(points={{-20,50},{0,50}},color={0,127,255}));
-  connect(weaBus.TWetBul,cooTowPar.TWetBul)
-    annotation (Line(points={{-70,90},{-70,44},{-2,44}},color={255,204,51},thickness=0.5));
-  connect(swi.y,pum.m_flow_in)
-    annotation (Line(points={{51,-80},{60,-80},{60,-128},{-80,-128},{-80,70},{-30,70},{-30,62}},color={0,0,127}));
-  connect(onOffCon.y,cooTowPar.on[1])
-    annotation (Line(points={{11,-80},{20,-80},{20,-130},{-100,-130},{-100,72},{-14,72},{-14,56},{-2,56}},color={255,0,255}));
-  connect(onOffCon.y,cooTowPar.on[2])
-    annotation (Line(points={{11,-80},{20,-80},{20,-130},{-100,-130},{-100,72},{-14,72},{-14,56},{-2,56}},color={255,0,255}));
+  connect(TSetLea.y, conFan.u_s)
+    annotation (Line(
+      points={{-39,-10},{-32,-10}},
+      color={0,0,127}));
+  connect(tow.TLvg, conFan.u_m)
+    annotation (Line(
+      points={{43,-47},{54,-47},{54,-30},{-20,-30},{-20,-22}},
+      color={0,0,127}));
+  connect(onOffCon.y, tow.on[1])
+    annotation (Line(points={{1,-190},{12,-190},{12,
+          -236},{-96,-236},{-96,-34},{12,-34},{12,-44},{20,-44}}, color={255,0,255}));
+  connect(onOffCon.y, tow.on[2])
+    annotation (Line(points={{1,-190},{12,-190},{12,
+          -236},{-96,-236},{-96,-34},{12,-34},{12,-44},{20,-44}}, color={255,0,255}));
+  connect(conFan.y, tow.uFanSpe)
+    annotation (Line(points={{-9,-10},{4,-10},{4,-48},
+          {20,-48}}, color={0,0,127}));
+  connect(weaBus.TWetBul, tow.TWetBul)
+   annotation (Line(
+      points={{-60,50},{-4,50},{-4,-56},{20,-56}},
+      color={255,204,51},
+      thickness=0.5), Text(
+      string="%first",
+      index=-1,
+      extent={{-6,3},{-6,3}},
+      horizontalAlignment=TextAlignment.Right));
```

Comment:
Change annotations to one line.

Reply:
Done. The last one I seperated into two lines.